README.md

```diff
@@ -1,5 +1,205 @@
+
+## Quick Start
+
+```bash
+# Setup
+git clone https://github.com/X-lab2017/open-digger-mcp-server.git
+cd open-digger-mcp-server && cd mcp-server
+npm install
+npm run build
+
+# Start server
+npm start
+```
+
+> [!IMPORTANT]  
+> Don't forget to configure Cursor (update path in .cursor/mcp.json)
+
+
+Expected output:
+
+```
+OpenDigger MCP Server running (on stdio)...
+```
+
+<br/>
+
+> [!IMPORTANT]  
+> If you are using Cursor AI IDE, you should see the MCP server (**`open-digger-mcp`**) toast in the bottom-left corner. You should _**enable**_ it, & now you can start using the tools and prompts provided by the MCP server. To verify, open Cursor Settings and check the MCP Servers section → you should see `open-digger-mcp` listed there. 
+
+
+![opendigger-mcp-img](https://res.cloudinary.com/dmlwye965/image/upload/v1758830133/open-digger-mcp-cursorSettings-snap_eewkz6.png)
+
+
+To further confirm that the server is functioning correctly, you can check the following indicators in Cursor:
+
+- ✅ **Green dot** next to "open-digger-mcp" title
+- ✅ **"6 tools"** displayed in server status
+- ✅ **"3 prompts"** displayed in server status
+- ✅ **No error messages** or red indicators
+
+<br/>
+<div style="display: flex; justify-content: center;">
+  <div style="width: 500px; height: 300px; border: 2px solid #ccc; border-radius: 8px; padding: 10px; display: flex; justify-content: center; align-items: center;">
+    <img src="https://res.cloudinary.com/dmlwye965/image/upload/v1758836416/demo-mcp-opendigger-gif_abn1yq.gif" alt="demo-mcp-opendigger" style="max-width: 100%; max-height: 100%;">
+  </div>
+</div>
+<br/>
+
+
+> [!TIP]
+> Please refer to [`Installation.md`](./INSTALLATION.md) for detailed installation instructions and configuration options for different IDEs.
+
+---
+
+## Features
+
+### Tools (6 Available)
+
+| No. | Tool                                | Description                                      |
+|-----|-------------------------------------|--------------------------------------------------|
+| 1   | **`get_open_digger_metric`**        | Fetch single repository metrics                  |
+| 2   | **`get_open_digger_metrics_batch`** | Batch operations for multiple metrics            |
+| 3   | **`compare_repositories`**          | Multi-repository comparative analysis            |
+| 4   | **`analyze_trends`**                | Growth trend analysis over time periods          |
+| 5   | **`get_ecosystem_insights`**        | Ecosystem analytics & insights                   |
+| 6   | **`server_health`**                 | System diagnostics and health monitoring (Beta) |d
+
+
+### Prompts (3 Available)
+1. **`repo_health_analysis`** - Comprehensive repository health reports
+2. **`repo_comparison`** - Competitive repository analysis
+3. **`developer_insights`** - Developer activity and contribution analysis
+
+
+### Metrics
```

Comment:
Need to synchronize with the code of index.ts.

Reply:
Sure thing, I've updated it to this,


- **Core Metrics**: `openrank`, `stars`, `forks`, `contributors`, `participants`, `issues_new`, `issues_closed`, `pull_requests`, `commits`, `activity`
- **Extended Metrics**: `technical_fork`, `bus_factor`, `releases`, `inactive_contributors`, `maintainer_count`, `community_activity`
- **Additional Metrics**: `change_requests`, `pull_requests_accepted`, `pull_requests_merged`, `issue_comments`, `issue_response_time`, `code_change_lines`, `developer_network`


So, it should be aligned now with `index.ts`. Thanks!